.ci/scripts/lint-commits.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
+set -xeo pipefail
 
-set -xe
+export PATH=${PATH}:/$(pwd)/node_modules:/$(pwd)/node_modules/.bin
+export HOME=$(pwd)
+
+npm install
```

Comment:
I didn't mind you just installing the module you needed. I just didn't know if you _also_ installed all the other modules, in which case it didn't make any sense. So feel free to swap it back.

Reply:
there is not too much difference a few seconds so it is fine to make npm install

Reply:
Doing an `npm install` every time we run the linter seems a bit weird to me. 🤔

Reply:
this was my first proposal, I've changed, now, Should I change it again to the first approach?